nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -611,7 +611,7 @@ define([], () => {
                     },
                 },
                 actionButton: {
-                    name: 'reRunApp',
+                    name: 'resetApp',
```

Comment:
since `resetApp` and `rerunApp` do basically the same thing, I'm merging them into a single entity known as `resetApp`